webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTableHelpers.js

```diff
@@ -30,18 +30,39 @@ const buildTableRow = (subscription, availableQuantities, updatedQuantity) => {
       quantity: updatedQuantity[subscription.id],
     };
   }
+
   return baseSubscription;
 };
 
+const buildTableCollapseRow = (subscriptionGroup) => {
+  const first = subscriptionGroup.subscriptions[0];
+  const heading = {
+    id: 0,
+    collapsible: true,
+    contract_number: 'NA',
+    start_date: 'NA',
+    end_date: 'NA',
+    consumed: 'NA',
+    product_id: first.product_id,
+    name: first.name,
+    virt_only: first.virt_only,
+    hypervisor: first.hypervisor,
+  };
+  return heading;
+};
+
 const buildTableRowsFromGroup = (subscriptionGroup, availableQuantities, updatedQuantity) => {
   const { open, subscriptions } = subscriptionGroup;
+  const rows = [];
```

Comment:
I think rows can be initialized inside the below if statement, so that we don't initialize it unnecessarily